docs/topics/gradle/gradle-backend-tutorial.md

```diff
@@ -0,0 +1,115 @@
+[//]: # (title: Get started with Gradle and Kotlin/JVM)
```

Comment:
I'd also rename the file to "get-started-with-jvm-gradle-project" or something similar

Reply:
What do you think?
I believe that "backend" is not necessarily mentioned here

Reply:
I like your proposal:)